docs/user_guide/models/multivi.rst

```diff
@@ -0,0 +1,228 @@
+    p_{nj} &= g_z^j\left(z_n, s_n\right) \\
+     \ell_n &= f_\ell\left(y_n\right)\\
+     y_{nj} &\sim \mathrm{Bernoulli}\left(p_{nj} \cdot \ell_n \cdot r_j \right)
+    \end{align}
+
+For RNA-seq, the prior parameters :math:`\ell_\mu` and :math:`\ell_{\sigma^2}` are computed per batch as the mean and
+variance of the log library size over cells. The expression data are generated from a count-based likelihood
+distribution, which here, we denote as the :math:`\mathrm{ObservationModel}`. While by default the
+:math:`\mathrm{ObservationModel}` is a :math:`\mathrm{ZeroInflatedNegativeBinomial}` (ZINB) distribution parameterized
+by its mean, inverse dispersion, and non-zero-inflation probability, respectively, users can pass
+``gene_likelihood = "negative_binomial"`` to :class:`~scvi.model.MultiVI`, for example, to use a simpler
+:math:`\mathrm{NegativeBinomial}` distribution.
+
+For ATAC-seq, detecting a region as accessible (:math:`y_{nj} > 0`) is
+generated by a Bernoulli random variable which depends on a cell-specific latent variable :math:`z_n`, which captures
+biological heterogeneity, and two auxiliary scaling factors :math:`\ell_n, r_j`, which account for cell-specific and
+region-specific technical effects, respectively.
+
+
+
+The generative process of MultiVI uses neural networks to produce:
+
+.. math::
+   :nowrap:
+
+   \begin{align}
+      f_w(z_n, s_n) &: \mathbb{R}^{d} \times \{0, 1\}^K \to \Delta^{G-1}\\
+      f_h(z_n, s_n) &: \mathbb{R}^d \times \{0, 1\}^K \to (0, 1)^T\\
+      g_z(z_n, s_n) &: \mathbb{R}^{d} \times \{0, 1\}^K \to \left[0,1\right]^M
+   \end{align}
+
+which respectively decode the denoised gene expression, non-zero-inflation probability (only if using ZINB) and
+estimates the probability of accessibility.
+
+The latent variables, along with their description are summarized in the following table:
+
+.. list-table::
+   :widths: 20 90 15
+   :header-rows: 1
+
+   * - Latent variable
+     - Description
+     - Code variable (if different)
+   * - :math:`z_n \in \mathbb{R}^d`
+     - Low-dimensional representation capturing the state of a cell.
+     - N/A
+   * - :math:`\rho_n \in \Delta^{G-1}`
+     - Denoised/normalized gene expression.
+     - ``px_scale``
+   * - :math:`\ell_n \in (0, \infty)`
+     - Library size for RNA.
+     - N/A
+   * - :math:`\theta_g \in (0, \infty)`
+     - Inverse dispersion for negative binomial. This can be set to be gene/batch specific for example (and would thus be :math:`\theta_{kg}`), by passing ``dispersion="gene-batch"`` during model intialization. Note that ``px_r`` also refers to the underlying real-valued torch parameter that is then exponentiated on every forward pass of the model.
+   * - :math:`p_r`
+     - Accessibility probability estimate
+   * - :math:`\ell_n \in \left[0,1\right]`
+     - Cell-wise scaling factor
+     - ``d``
+   * - :math:`r_j \in \left[0,1\right]`
+     - Region-wise scaling factor
+     - ``f``
```

Comment:
```suggestion
   * - :math:`\theta_g \in (0, \infty)`
     - Inverse dispersion for negative binomial. This can be set to be gene/batch specific for example (and would thus be :math:`\theta_{kg}`), by passing ``dispersion="gene-batch"`` during model intialization. Note that ``px_r`` also refers to the underlying real-valued torch parameter that is then exponentiated on every forward pass of the model.
     - N/A
   * - :math:`p_r`
     - Accessibility probability estimate
     - N/A
   * - :math:`\ell_n \in \left[0,1\right]`
     - Cell-wise scaling factor
     - ``d``
   * - :math:`r_j \in \left[0,1\right]`
     - Region-wise scaling factor
     - ``f``
```